docs/src/appendix/forge-library/signature/stark_curve.md

```diff
@@ -0,0 +1,78 @@
+# `STARK Curve`
+
+This module contains the implementation of `KeyPairTrait` for the [STARK Curve](https://docs.starknet.io/documentation/architecture_and_concepts/Cryptography/stark-curve/).
+
+> ⚠️ **Security Warning**
+>
+> Please note that cryptography in Starknet Foundry is still experimental and **has not been audited yet**.
```

Comment:
```suggestion
> Please note that cryptography in Starknet Foundry is still experimental and **has not been audited**.
```
I don't think it will be soon, let's not suggest that.